### PR TITLE
fix advancing frames not working

### PR DESCRIPTION
**Pull request by tybug**
Previously clicking advance frame works once and then don't work again. Same thing with retreat frame. This somehow fixes both for me, I'm not sure why.

### CODE REVIEW

**Comment by tybug** [comment]
Don't like either of those. We shouldn't rely on the sampling rate of the replays being identical, and as you say, looking only at the first replay is bound to cause problems. 

There's no reason why we can't just fix the damn issue. I'll try and reproduce your gif.

File: circleguard/visualizer.py [hunk 1]
```diff
@@ -517,7 +519,11 @@ def __init__(self, replays=[], beatmap_path=""):
         self.slider.setFixedHeight(20)
         self.slider.setStyleSheet("outline: none;")
         self.renderer.update_signal.connect(self.update_slider)
-        self.slider.valueChanged.connect(self.renderer.seek_to)
+        # don't want to use valueChanged because we change the value
+        # programatically and valueChanged would cuase a feedback loop.
+        # sliderMoved only activated on true user action, where we actually
```
Comment (InvisibleSymbol) [inline circleguard/visualizer.py]:
```suggestion
        # sliderMoved only activates on true user action, when we actually
```
unsure about the where/when part

File: circleguard/visualizer.py [hunk 2]
```diff
@@ -517,7 +519,11 @@ def __init__(self, replays=[], beatmap_path=""):
         self.slider.setFixedHeight(20)
         self.slider.setStyleSheet("outline: none;")
         self.renderer.update_signal.connect(self.update_slider)
-        self.slider.valueChanged.connect(self.renderer.seek_to)
+        # don't want to use valueChanged because we change the value
+        # programatically and valueChanged would cuase a feedback loop.
```
Comment (InvisibleSymbol) [inline circleguard/visualizer.py]:
```suggestion
        # programatically and valueChanged would cause a feedback loop.
```

**Comment by tybug** [comment]
thanks for catching those, you were right on both accounts

**Pull request merged**